README.md

```diff
@@ -12,41 +12,6 @@ The server consists of three components:
 The main target environment is node, nevertheless, all components are implemented in an ismorphic fashion and also provide
 an entrypoint to target browser environments (e.g. running the server in a web worker)
 
-## Integration & Features
```

Comment:
Should we add some information about the websocket server in `## Start & Debug`?